src/drivers/optical_flow/paa3905/PAA3905.hpp

```diff
@@ -121,6 +121,7 @@ class PAA3905 : public device::SPI, public I2CSPIDriver<PAA3905>
 	bool _data_ready_interrupt_enabled{false};
 
 	uint32_t _scheduled_interval_us{SAMPLE_INTERVAL_MODE_0 / 2};
+	uint32_t _backup_schedule_interval_us{200000};
```

Comment:
We're trying to embrace google style guide (mostly). 

```suggestion
	static constexpr uint32_t kBackupScheduleIntervalUs{200_ms};
```

Reply:
Forced pushed